DNN Platform/Website/admin/Security/PasswordReset.ascx.cs

```diff
@@ -1,362 +1,368 @@
+            ClientResourceManager.RegisterScript(this.Page, "~/Resources/Shared/scripts/dnn.jquery.tooltip.js");
+            ClientResourceManager.RegisterScript(this.Page, "~/Resources/Shared/scripts/dnn.PasswordStrength.js");
+            ClientResourceManager.RegisterScript(this.Page, "~/DesktopModules/Admin/Security/Scripts/dnn.PasswordComparer.js");
+
+            ClientResourceManager.RegisterStyleSheet(this.Page, "~/Resources/Shared/stylesheets/dnn.PasswordStrength.css", FileOrder.Css.ResourceCss);
+
+            if (this.PortalSettings.LoginTabId != -1 && this.PortalSettings.ActiveTab.TabID != this.PortalSettings.LoginTabId)
+            {
+                this.Response.Redirect(this._navigationManager.NavigateURL(this.PortalSettings.LoginTabId) + this.Request.Url.Query);
+            }
+
+            this.cmdChangePassword.Click += this.cmdChangePassword_Click;
+
+            this.hlCancel.NavigateUrl = this._navigationManager.NavigateURL();
+
+            if (this.Request.QueryString["resetToken"] != null)
+            {
+                this.ResetToken = this.Request.QueryString["resetToken"];
+                this.txtUsername.Enabled = false;
+            }
+
+            var useEmailAsUserName = PortalController.GetPortalSettingAsBoolean("Registration_UseEmailAsUserName", this.PortalId, false);
+            if (useEmailAsUserName)
+            {
+                this.valUsername.Text = Localization.GetString("Email.Required", this.LocalResourceFile);
+            }
+            else
+            {
+                this.valUsername.Text = Localization.GetString("Username.Required", this.LocalResourceFile);
+            }
+
+            if (this.Request.QueryString["forced"] == "true")
+            {
+                this.lblInfo.Text = Localization.GetString("ForcedResetInfo", this.LocalResourceFile);
+            }
+
+            this.txtUsername.Attributes.Add("data-default", useEmailAsUserName ? this.LocalizeString("Email") : this.LocalizeString("Username"));
+            this.txtPassword.Attributes.Add("data-default", this.LocalizeString("Password"));
+            this.txtConfirmPassword.Attributes.Add("data-default", this.LocalizeString("Confirm"));
+            this.txtAnswer.Attributes.Add("data-default", this.LocalizeString("Answer"));
+
+            if (!this.Page.IsPostBack)
+            {
+                this.LoadUserInfo();
+            }
+        }
+
+        protected override void OnPreRender(EventArgs e)
+        {
+            base.OnPreRender(e);
+            if (!string.IsNullOrEmpty(this.lblHelp.Text) || !string.IsNullOrEmpty(this.lblInfo.Text))
+            {
+                this.resetMessages.Visible = true;
+            }
+
+            if (Host.EnableStrengthMeter)
+            {
+                this.passwordContainer.CssClass = "password-strength-container";
+                this.txtPassword.CssClass = "password-strength";

```

Comment:
These classes probably should be removed from the `ascx` file so that they're applied conditionally. Currently, they're already in the `ascx` file so these lines have no effect.

Reply:
Good point.  Thanks @bdukes.